CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Add MC Zot deployment as a registry mirror for `docker.io` and `gsoci.azurecr.io` registries. This is the new default behavior.
+- Add `gsoci.azurecr.io` as a mirror for `docker.io`.
```

Comment:
This needs to be updated.

Reply:
Updated 👍

Reply:
Do I get something wrong? `gsoci.azurecr.io` is not mirroring `docker.io`. It contains our images after a certain point (about 1 year ago). And the MC zot deployment is only a mirror for `gsoci.azurecr.io` at the moment.

Looking at your example here: https://github.com/giantswarm/cluster/pull/260#issue-2400412416 that looks good. But then this change log is not correct.

Reply:
I have updated the changelog, now it just has this:

```
### Changed

- Add MC Zot deployment as a registry mirror for `gsoci.azurecr.io` registry. This is the new default behavior.

### Added

- Tests for checking rendered containerd config.
```